build-output-api/README.md

```diff
@@ -16,6 +16,7 @@ that are listed below.
 - [Edge Functions](./edge-functions)
 - [Edge Middleware](./edge-middleware)
 - [Image Optimization](./image-optimization)
+- [On-demand Incremental Static Regeneration (ISR)](./on-demand-isr)
```

Comment:
```suggestion
- [On-Demand Incremental Static Regeneration (ISR)](./on-demand-isr)
```
Seems we have some inconsistency with the case of the `demand` but I think it should be capitalized as that's how title.sh handles it, will open docs PR to update.